modules/base/shaders/billboardpoint_gs.glsl

```diff
@@ -71,30 +66,15 @@ const vec2 corners[4] = vec2[4](
 const int RenderOptionCameraViewDirection = 0;
 const int RenderOptionCameraPositionNormal = 1;
 
-
 void main() {
-  ta = 1.0;
   vec4 pos = gl_in[0].gl_Position;
-  gs_colorMap = colorMap[0];
-
-  double unit = PARSEC;
-
-  // Must be the same as the enum in RenderableBillboardsCloud.h
-  if (pos.w == 1.0) { unit = 1E3; }
-  else if (pos.w == 2.0) { unit = PARSEC; }
-  else if (pos.w == 3.0) { unit = 1E3 * PARSEC; }
-  else if (pos.w == 4.0) { unit = 1E6 * PARSEC; }
-  else if (pos.w == 5.0) { unit = 1E9 * PARSEC; }
-  else if (pos.w == 6.0) {
-    // Conversion factor from Parsecs to GigalightYears
-    unit = 306391534.73091 * PARSEC;
-  }
+  gs_colorParameter = colorParameter[0];
 
-  dvec4 dpos = modelMatrix * dvec4(dvec3(pos.xyz) * unit, 1.0);
+  dvec4 dpos = modelMatrix * dvec4(dvec3(pos.xyz), 1.0);
 
-  float scaleMultiply = exp(scaleFactor * 0.10);
+  float scaleMultiply = exp(scaleExponent);
```

Comment:
Thought when writing the docs for this:   Is it worthwhile to change this computation to be `10^exponent` rather than `e^exponent`?  Would be a bit more intuitive to explain, I think. 

Thoughts @alexanderbock @Roxeena ? :)

Reply:
Mathematically they are different so which one is being used? But the 10^exponent is more intuitive I think.

Reply:
Currently e^exponent is used (that's what the `exp` function does)

Changng it would involve recomputing all the exponents in the asset files, but I do think it would make a lot more sense

Reply:
10^exponent makes more sense to me, yea

Reply:
Fixed